aten/src/ATen/native/Distance.cpp

```diff
@@ -0,0 +1,11 @@
+#include "ATen/ATen.h"
+#include "ATen/NativeFunctions.h"
+
+
+namespace at { namespace native {
+
+Tensor pairwise_distance(const Tensor& x1, const Tensor& x2, double p, double eps) {
+  auto diff = abs(x1 - x2 + eps);
```

Comment:
I don't really understand why `eps` is necessary, but it was in the original code... do you know why?